tests/foreman/cli/test_errata.py

```diff
@@ -1164,42 +1172,36 @@ def rh_repo_module_manifest(module_entitlement_manifest_org, module_target_sat):
     return rh_repo
 
 
-"""Section for tests using RHEL7.7 Content Host.
+"""Section for tests using RHEL7/8 Content Hosts.
```

Comment:
shouldn't we include EL9 as well?
```suggestion
"""Section for tests using RHEL Content Hosts.
```

Reply:
We can do for RHEL9 as well, I would need to implement `errata_host` importing fixture `rhel_contenthost`, instead of just `rhel8_contenthost`, (and parameterized to desired version, 7, 8, 9 etc, for each test using `errata_host`). I will give this a try locally, as I've done this parameterized fixture setup for api already.